### DISCUSSION TITLE
Unable to execute HTTP request: Connect to IP:Port [/IP] failed: connect timed out

### DISCUSSION THREAD

**Discussion by cskr006** [aws/aws-sdk-java#2822] [Q&A]
Getting this error in AWS
What could be the possible reason or fix for this ?
using AWS SDK - 1.11.979

**Comment by yasminetalby** [comment]
Hello @cskr006 ,
Thank you very much for your message.
We would need more details to offer guidance.
Could you please provide any code sample/error logs/context?

Best,

Yasmine
